script.js

```diff
@@ -1,4 +1,149 @@
+// Requirements
+// 1. There are 2 players and players take turns.
+// 2. When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// 3. The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// 4. After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+//Define global variables
+//Checks if game mode is to roll dice or if user needs to specify order
+var GAME_MODE_ROLL_DICE = `dice Roll`;
+var GAME_MODE_CHOOSE_ORDER = `choose Order`
```

Comment:
good use of constants, but we don't have to camelCase the actual string values. They are actual values and should be cased however the string should look like